plugins/Denizen/scripts/cooking/Cutting/knifes.dsc

```diff
@@ -0,0 +1,39 @@
+paring_knife:
+    type: item
+    display name: <&f>Paring Knife
+    material: stone_sword
+    flags:
+        cooking_knife: true
+        accuracy: 8
+        size: 1
+        specialty: all
+
+chinese_cleaver:
+    type: item
+    display name: <&f>Chinese Cleaver
+    material: iron_sword
+    flags:
+        cooking_knife: true
+        accuracy: 4
+        size: 7
+        specialty: all
+
+chef_knife:
+    type: item
+    display name: <&f>Chef Knife
+    material: iron_sword
+    flags:
+        cooking_knife: true
+        accuracy: 5
+        size: 5
+        specialty: all
```

Comment:
Chef knives are good at vegetables and paring knives are good at meat?

Reply:
no they have specialties and paring knives just have better accuracy

Reply:
oh